index.js

```diff
@@ -61,7 +61,8 @@ function getUrlFromUsername( site, username ) {
         dribbble: 'dribbble.com',
         facebook: 'facebook.com',
         angellist: 'angel.co',
-        bitbucket: 'bitbucket.org'
+        bitbucket: 'bitbucket.org',
+        skype: 'none'
```

Comment:
Instead of this, how about

```javascript
skype: 'skype'
```

And modify the switch statement to 
```javascript
case 'skype':
  return url_map[site] + ':' + username + '?call'
```